noetic/ros-opencog/Dockerfile

```diff
@@ -0,0 +1,167 @@
+#
+# Container for ROS+OpenCog
+#
+# To build:
+# sudo docker build -t opencog/ros-noetic-opencog .
+#
+FROM opencog/ros-noetic-base
+MAINTAINER Linas Vepštas "linasvepstas@gmail.com"
+
+# Install required packages
+ENV LAST_OS_UPDATE 2020-04-23_6
+RUN apt-get -y update
+RUN apt-get -y upgrade
+# Base stuff for cogutil. All on one line, to avoid error
+# `Cannot create container with more than 127 parents`
+RUN apt-get -y install gcc g++ cmake binutils-dev libiberty-dev \
+    libboost-dev libboost-date-time-dev libboost-filesystem-dev \
+    libboost-program-options-dev libboost-regex-dev \
+    libboost-serialization-dev libboost-system-dev libboost-thread-dev \
+    cxxtest
+
+# Additional stuff needed to build and run relex.
+RUN apt-get -y install wordnet-dev wordnet-sense-index \
+    ant libcommons-logging-java libgetopt-java
+
+# Additional stuff for the AtomSpace: cython and guile and ODBC.
+# Additional stuff for OpenCog. We need telnet as a debug utility.
+# Need wget to download link-grammar source.
+# Need unzip to unzip JWNL.
+RUN apt-get -y install guile-2.2-dev cython unixodbc-dev odbc-postgresql \
+    wget telnet locales rlwrap unzip
+
+# -------------------------------------------------------------
+# Right now, just clone the OpenCog sources. They will be built
+# later on.
+WORKDIR /opencog
+RUN git clone https://github.com/opencog/cogutil.git
+RUN git clone https://github.com/opencog/atomspace.git
+RUN git clone https://github.com/opencog/lg-atomese.git
+RUN git clone https://github.com/opencog/opencog.git
+RUN git clone https://github.com/opencog/relex.git
```

Comment:
Please add three more to this list -- `atomspace-rocks`, (RocksDB backend), `cogserver` (needed for network I/O) and `atomspace-cog` (for a network-distributed atomspace).